### ISSUE TITLE
Editing properties on saved entities duplicates it

### ISSUE THREAD

**Issue by Janchorizo** [providedh/collaborative-platform#121]
Branch: develop
File: group_0_long_annotated_0_modified.xml
Steps to reproduce: Send  websocket message with the following payload
```
{"method":"modify","payload":[{"element_type":"entity_property","method":"PUT","edited_element_id":"person-2","old_element_id":"birth","parameters":{"birth":"2001-02-04"}}]}
```

The modified entity gets twice the attribute:
```
deleted: false
properties:
0: {name: "age", value: "child", saved: true, deleted: false}
1: {name: "birth", value: "2001-02-03", saved: true, deleted: true}
2: {name: "birth", value: "2001-02-04", saved: false, deleted: false}
3: {name: "forename", value: "Jerry", saved: true, deleted: false}
4: {name: "name", value: "Jerry Maguire", saved: true, deleted: false}
5: {name: "sex", value: "M", saved: true, deleted: false}
6: {name: "surname", value: "Maguire", saved: true, deleted: false}
resp: "#annotator-1"
saved: true
type: "person"
xml:id: "person-2"
```

**Comment by bug-rancher** [comment]
And that is ok. User can see result of modification - the old value (with parameters `saved: true, deleted: true`), and the new one (with parameters `saved:false, deled: false`). This concept is used in tags in xml too. When user save this change, old property disappear from response. It's not a bug, it's a feature.

**Comment by Janchorizo** [comment]
Please document this kind of things. Because it is very difficult to make a consistent interface if not.
If have taken any other decision that I should be aware of please include it in the docs.
Information regarding [unsaved | edited | saved] properties was only added for "un-listable" entities
and after mentioning I didn't understand the responses; so please fill any other aspect you feel relevant.

**Comment by bug-rancher** [comment]
You are right, my documentation should be better. I tried to implement Annotator's API in the way, that despite these are different kinds of elements (tags in xml, various models in database), every kind of element can be handled in this same way from frontend's point of view.

When we talked about purpose of attributes with "Added" and "Deleted" suffixes in xml content I always have in mind that this same concept (just with "saved" and "deleted" fields) is applied to whole backend's response, so to tags, certainties, entities and entities properties too. My bad. I updated point 3.1. in documentation.
